tests/geometry/test_tools.py

```diff
@@ -456,12 +456,13 @@ def test_hull_around_two_circles_xy_plane(self):
         bounding_box = hull.bounding_box
         assert bounding_box.z_min == bounding_box.z_max == 0
 
-    @pytest.mark.parametrize("bad_plane", ["ab", "", None, ["x", "y"]])
+    @pytest.mark.parametrize("bad_plane", ["ab", "", ["x", "y"]])
```

Comment:
refined the test cases in PR #3036, commit 0a4af30. Made two tests, one for ValueError (for "ab", "") and other for TypeError (for None, ["x", "y"]). Because of the Enumification, ".upper()" on wrong type is not possible so a TypeError is raised instead. See tools.py